src/index.tsx

```diff
@@ -5,6 +5,7 @@ export type CarouselItem = {
 	readonly alt?: string;
 	readonly content: React.ReactNode;
 	readonly image: string;
+	readonly click?: () => void;
```

Comment:
Can we please rename this to `onClick` to indicate that it is actually an action?

Reply:
surething